modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/constants/RemoteAppAdminFDSNames.java

```diff
@@ -17,9 +17,8 @@
 /**
  * @author Bruno Basto
  */
-public class RemoteAppAdminConstants {
+public class RemoteAppAdminFDSNames {
```

Comment:
Not sure about this rename. Is it a current pattern?

Anyway let's try with it

Reply:
Yeah, that's the naming pattern Brian requested in https://github.com/brianchandotcom/liferay-portal/pull/111833#issuecomment-1009551559